test/example_bot.gleam

```diff
@@ -28,8 +27,11 @@ pub fn main(token: String, client_id: String) {
       ],
     )
 
-  discord_gleam.wipe_slash_commands(bot, client_id)
-  discord_gleam.register_commands(bot, client_id, [test_cmd])
+  discord_gleam.wipe_global_commands(bot, client_id)
+  discord_gleam.register_global_commands(bot, client_id, [test_cmd])
+
+  discord_gleam.wipe_guild_commands(bot, client_id, guild_id)
+  discord_gleam.register_guild_commands(bot, client_id, guild_id, [test_cmd])
```

Comment:
Here we could probably have two diffrent commands like test and test2

Reply:
doesnt matter :sippycup: